.github/workflows/update-flack-lock.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: update-flake-lock
+on:
+  workflow_dispatch:
+  schedule:
+    # runs weekly on Saturday at 00:00
+    - cron: "0 0 * * 6"
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install Nix
+        uses: DeterminateSystems/nix-installer-action@main
```

Comment:
What is the advantage of this nix installer? We use the cachix nix installer in our other nix workflow: https://github.com/teslamate-org/teslamate/blob/93cc34437c6466c993fde98bb271faaa9ed8a58d/.github/workflows/ensure_linting.yml#L16

Reply:
Good question. I hadn't noticed that there are multiple competing solutions.

Perhaps I should change it to the one we already use.

I see a comparison at https://discourse.nixos.org/t/which-github-nix-installer-action-is-faster/25878

Reply:
Supposedly https://github.com/nixbuild/nix-quick-install-action is fastest, it is single user only. But I think we would need to benchmark this before switching.

Reply:
Yeah, I found the above-mentioned benchmark after deciding for the cachix installer, which still claims to be fastest in most cases.